doc/getting_started/index.rst

```diff
@@ -330,38 +330,24 @@ Set up Reference UOS
 Device Manager memory allocation mechanism
 ==========================================
 
-There are two Device Manager memory allocation mechanisms available:
+There only provides Huge Page Tables (HugeTLB) Device Manager memory allocation mechanisms:
 
-- Contiguous Memory Allocator (CMA), and 
-- Huge Page Tables (HugeTLB).  HugeTLB is the default.
+To support HugeTLB, need to do huge page reservation:
 
-To choose CMA, do the following:
+  - For 1G huge page reservation, add ``hugepagesz=1G hugepages=reserved_pg_num``
+    (for example, ``hugepagesz=1G hugepages=4``) to the SOS cmdline in
+    ``acrn.conf`` (for EFI)
 
-1) Add ``cma=reserved_mem_size@recommend_memory_offset-0``, (for example
-   ``cma=2560M@0x100000000-0``) to the SOS cmdline in ``acrn.conf``
+  - For 2M huge page reservation, after the SOS starts up, run the
+    command::
 
-2) Start ``acrn-dm`` *without* the ``-T`` option
+      echo reserved_pg_num > /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages
 
-To support HugeTLB, do the following:
-
-1) Do huge page reservation
-
-   - For 1G huge page reservation, add ``hugepagesz=1G hugepages=reserved_pg_num``
-     (for example, ``hugepagesz=1G hugepages=4``) to the SOS cmdline in
-     ``acrn.conf`` (for EFI)
-
-   - For 2M huge page reservation, after the SOS starts up, run the
-     command::
-
-        echo reserved_pg_num > /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages
-
-   .. note::
-      You can use 2M reserving method to do reservation for 1G page size, but it
-      may fail.  For an EFI platform, you may skip 1G page reservation
-      by using a 2M page, but make sure your huge page reservation size is
-      large enough for your usage.
-
-2)  Start ``acrn-dm`` *with* the ``-T`` option.
+  .. note::
+     You can use 2M reserving method to do reservation for 1G page size, but it
+     may fail.  For an EFI platform, you may skip 1G page reservation
+     by using a 2M page, but make sure your huge page reservation size is
+     large enough for your usage.
```

Comment:
@dbkinder , do you want to review it now or address potential changes in a subsequent PR (that you or I could create). I'm talking about the changes in the Geting Started Guide.

Reply:
I find this note technically confusing.  @gvancuts can you suggest a change to clarify?